cudamapper/src/index.cu

```diff
@@ -64,6 +65,49 @@ std::unique_ptr<IndexHostCopyBase> IndexHostCopyBase::create_cache(const Index&
                                            cuda_stream);
 }
 
+std::unique_ptr<IndexDescriptorBase> create_index_descriptor(read_id_t first_read,
```

Comment:
Why not just use the `std::make_unique<IndexDescriptor>(first_read, number_of_reads);`, directly in the `group_reads_into_descriptors()` function?

Reply:
removed in favor of using the class directly.